src/Mvc/test/Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.Link.html

```diff
@@ -23,13 +23,13 @@
     <link href="/styles/site.css" rel="stylesheet" />
 
     <!-- Globbed link tag with missing file -->
-
+    
```

Comment:
Thanks @BrennanConroy. This is what I get for editing generated files manually (unable to regen when I was mucking with these two files).